src/sentry/notifications/notification_action/grouptype.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from sentry import features
+from sentry.issues.grouptype import GroupCategory, GroupType
+from sentry.models.organization import Organization
+from sentry.ratelimits.sliding_windows import Quota
+
+
+@dataclass(frozen=True)
+class SendTestNotification(GroupType):
+    type_id = 9888
+    slug = "send-test-notification"
+    description = "Send test notification"
+    category = GroupCategory.TEST_NOTIFICATION.value
+    released = False
+    in_default_search = False
+    enable_auto_resolve = False
+    enable_escalation_detection = False
+    enable_status_change_workflow_notifications = False
```

Comment:
IMO I think this can be left to the default (True) as well - if a user wants to play around with a test issue, we should support as much of the errors-like workflows as possible including notifications for resolution/regression